network_wrangler/RoadwayNetwork.py

```diff
@@ -116,3 +117,42 @@ def write(self, filename: str, path: str = '.') -> None:
 
         shapes_file = os.path.join(path, filename + "_shape.geojson")
         self.shapes_df.to_file(shapes_file, driver='GeoJSON')
+
+
+
+    def apply_roadway_feauture_change(self, project_card: str) -> bool:
+        '''
+        Changes the road network according to the project card passed
+        
+        args:
+        project_card: the path to the project card
+        
+        returns:
+        bool: True if successful.
+        '''
+        #import pdb; pdb.set_trace()
+        with open(project_card, 'r') as card:
+            card_dict = yaml.safe_load(card)
+            
+            road_dict = card_dict.get("Road")            
+            name_str = road_dict.get(list(road_dict)[0])
+            id = name_str.split('=')[1]
+            
+            change_dict = card_dict.get("Change")
+            existing_value = change_dict.get(list(change_dict)[0])
```

Comment:
existing value should be checked with the value in the network; 
optionally, a failed match would trigger a warning or full fail